<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scriptsMenu</class>
 <widget class="QWidget" name="scriptsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="autoModeButton">
      <property name="text">
       <string>Автоматический режим</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="downloadProfilesButton">
      <property name="text">
       <string>Скачивание найденных профилей</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="findNewAccountsButton">
      <property name="text">
       <string>Поиск новых аккаунтов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createRatingsFileButton">
      <property name="text">
       <string>Создание файлов с оценками</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cacheResetButton">
      <property name="text">
       <string>Сброс кэша</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fileRepairButton">
      <property name="text">
       <string>Починка файлов</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
